### DISCUSSION TITLE
Handling Case Sensitivity Issue in Supabase Function

### DISCUSSION THREAD

**Discussion by MachariaK** [supabase/supabase#19820] [Questions] [answered]
I'm working with Supabase and have a function defined as follows: CREATE OR REPLACE FUNCTION save_callback_data(Body JSON). However, when I run this function, it seems that Supabase is changing the parameter name from Body to body. This becomes problematic as the incoming JSON always contains a key named Body, leading to a conflict.

Is there a way to make the parameter case insensitive in the function definition, so that regardless of whether it's called Body or body, the function can handle it correctly? Keep in mind that the data is coming from a third party and will consistently have the key named Body. Any suggestions on how to resolve this case sensitivity issue in Supabase?

**Comment by GaryAustin1** [comment, marked as answer]
This is not a Supabase thing.  Postgres converts names to lower case UNLESS they have double quotes around them "MyName".
So your parameters both in the function definition and the SQL code it contains has to have double quotes around them.
